_dashboards/geojson-regionmaps.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Region map visualizations
+nav_order: 7
+---
+
+# Region map visualizations
+
+OpenSearch Dashboards provides basic map tiles with a standard vector map that you can use to create your region map visualizations. You can configure the base map tiles using the WMS map server.
+
+You can't configure a server to support user-defined vector map layers. However, you can configure your own GeoJSON file and upload it for this purpose. 
+{: .note}
+
+OpenSearch also provides a standard set of GeoJSON files to provide a vector map with your regional maps.
+
+## Custom vector maps with GeoJSON
+
+If you have a specific locale that is not provided by OpenSearch Dashboards vector maps, such as a US county or US zip code, you can create your own custom vector map with a GeoJSON file. To use this feature, you have to install both the [Dashboards-Maps frontend plugin](https://github.com/opensearch-project/dashboards-maps) and the [OpenSearch Geospatial backend plugin](https://github.com/opensearch-project/geospatial).
+
+GeoJSON format allows you to encode geographic data structures. To learn more about the GeoJSON specification, go to [geojson.org](https://geojson.org/).
+
+You can use [geojson.io](https://geojson.io/#map=2/20.0/0.0) to extract GeoJSON files.
+{: .tip}
+
+You can also create your own custom vector map by uploading a JSON file that contains GEO data for your customized regional maps. The JSON file contains vector layers for visualization.
+
+1. Prepare a JSON file to upload. Make sure the file has either .geojson or .json extension.
+1. On the top menu bar, go to **OpenSearch Dashboards > Visualize**.
+1. Select the **Create Visualization** button.
+1. Select **Region Map**.
+1. Choose a source, for example: **[Flights] Flight Log**.
+1. In the right panel, select **Import Vector Map**. 
+1. In **Upload map**, select or drag and drop your JSON file.
+Enter **Map name prefix** (for example, `us-county`). Your map will have the prefix that you defined followed by the `-map` suffix (for example, `us-county-map`). 
+Select the **Import file** button. 
+Once the upload is successful, you will see a popup prompting you to refresh the map. Select the **Refresh** button.
+
+<img src="{{site.url}}{{site.baseurl}}/images/import-vector-map.png" alt="import a Geo .json file" width="380"/>
+
+## Layer options
+
+If you upload a custom GeoJSON file, you can toggle between the layer options: **Default vector map** or **Custom vector map**.
+
+1. On the top menu bar, go to **OpenSearch Dashboards > Visualize**.
+1. Select the **Create Visualization** button.
+1. Select **Region Map**.
+1. Choose a source, for example: **[Flights] Flight Log**.
```

Comment:
Change to "Choose a source. For example,"

Reply:
Done